extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -75,8 +74,7 @@ export async function setLocalAppSetting(projectFolder: string, key: string, val
 	}
 
 	settings.Values[key] = value;
-	void fs.promises.writeFile(localSettingsPath, JSON.stringify(settings, undefined, 2));
-
+	fs.writeFileSync(localSettingsPath, JSON.stringify(settings, undefined, 2));
```

Comment:
You should avoid sync functions since they block execution, use async ones and await instead.